README.md

```diff
@@ -144,6 +144,10 @@ Specifies a set of CSS properties to apply to a given group highlight. Multiple
 
 Reload highlight configuration.
 
+#### `:hilite remove <group|pattern> <text>`
```

Comment:
this should probably be `:hilite remove <group> <pattern|text>` since hilites can be regexp

Reply:
`<group|pattern>` is intended to convey that this is either the literal `group` or `pattern` to indicate which type of object is being removed. `<text` is meant to be the key in the relevant map, so either the regex pattern or the group name. Seems like this should be wordsmithed to be more clear however.